Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
+abort 'Ruby should be >= 2.1.0' unless RUBY_VERSION.to_f >= 2.1
```

Comment:
I think we should also add this to the `.gemspec` file:

```ruby
  spec.required_ruby_version = '>= 2.1.0'
```

Which uses [RubyGem's Ruby version support](http://guides.rubygems.org/specification-reference/#required_ruby_version=).